csfieldguide/curriculum_guides/content/en/apcsp/sections/todo.md

```diff
@@ -4,99 +4,108 @@ Several learning objectives are not currently covered by the CS Field Guide.
 This "to-do" list includes general computer science topics and suggestions for how we can update the field guide to cover the learning objectives. Different levels of changes are suggested:
 
 - new chapter: some topics are best put into stand-alone new field guide chapters
- - Hardware Abstractions [Phase+]
- - Software Abstractions [Phase4]
- - Models and Simulation [Phase3]
- - Network Systems [Phase+]
- - Logic [Phase+]
+    - Hardware Abstractions [Phase+]
```

Comment:
We should probably make an issue to get this page updated. I don't think it's important enough to update in this PR.